src/Core/src/Platform/iOS/SizeExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.Maui;
+using SizeF = CoreGraphics.CGSize;
+
+namespace Microsoft.Maui
+{
+	public static class SizeExtensions
+	{
+		public static SizeF ToNative(this Size size)
```

Comment:
Maybe not right now, but all this is in essentials, so it might be worth using that... Maybe for P3?